README.md

```diff
@@ -6,17 +6,32 @@ Release automation for pandas.
   - [  ] `TAG` in `Makefile`
   - [  ] `GH_USERNAME` in `Makefile`
 
-If running for the first time be sure to initialize repos
+## **If running for the first time:**
 
+- Create a conda enviroment based on environment.yml:
+```sh
+conda env create 
+```
+- Activate the environment:
+```sh
+conda activate pandas-release
+```
+- Be sure you have the following forked on your github:
+  - https://github.com/pandas-dev/pandas-website
+  - https://github.com/conda-forge/pandas-feedstock
+  - https://github.com/MacPython/pandas-wheels
+  - https://github.com/pandas-dev/pandas   
+  
+  &nbsp;
```

Comment:
Is this required for anything?

Reply:
It introduces a newline. for better readability and clearer separation between the commands